thumbor/doctor.py

```diff
@@ -83,37 +158,63 @@ def check_compiled_extensions():
             print(cf.bold_green(f"{CHECK} {ext_name}"))
         except ImportError as error:
             print(cf.bold_red(f"{CROSS} {ext_name}"))
-            errors.append(error)
+            errors.append(
+                format_error(
+                    f"Extension {extension}",
+                    str(error),
+                    (
+                        "Extension could not be compiled. "
+                        "This will lead to filter being disabled."
+                    ),
+                )
+            )
 
     return errors
 
 
-def check_modules():
+def format_error(dependency, err, msg):
+    formatted_msg = "\n\t".join(msg.split("\n"))
+    result = f"""
+* {dependency}
+    Error Message:
+        {err}
+
+    Error Description:
+        { formatted_msg }
+    """
+    return result.strip()
+
+
+def check_modules(cfg):
     newline()
-    subheader("Verifying libraries support...")
     errors = []
 
-    modules = (
+    modules = [
         (
             "pycurl",
-            "Thumbor works much better with PyCurl. For more information visit http://pycurl.io/.",
-        ),
-        (
-            "cv2",
-            "Thumbor requires OpenCV for smart cropping. "
-            "For more information check https://opencv.org/.",
-        ),
-        (
-            "pyexiv2",
-            "Thumbor uses exiv2 for reading image metadata. "
-            "For more information check https://python3-exiv2.readthedocs.io/en/latest/.",
+            (
```

Comment:
I'm no python dev, but curious: why is `pycurl` formatted as is, and the next module `cairosvg` not the same?

Reply:
Not sure either. Fixing anyway... Good catch!